assets/training/finetune_acft_image/environments/acft_image_huggingface/context/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # PTCA image
-FROM mcr.microsoft.com/aifx/acpt/vision/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag:biweekly\.\d{6}\.\d+\.v\d+}}
+FROM mcr.microsoft.com/aifx/acpt/vision/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag:biweekly\.\d{6}\.\d{1}.*}}
```

Comment:
{{latest-image-tag:biweekly\.\d{6}\.\d{1}}}
should be fine to pick up second Oct update
https://mcr.microsoft.com/v2/aifx/acpt/vision/stable-ubuntu2004-cu117-py38-torch1131/tags/list(https://mcr.microsoft.com/v2/aifx/acpt/vision/stable-ubuntu2004-cu117-py38-torch1131/tags/list)

Reply:
It is picking "biweekly.202310.2". Did you mean something else, @vizhur ?

Reply:
it might get messed up with tags like         "biweekly.202309.1.v1",